packages/storybook-vue/src/stories/Link/argTypes.ts

```diff
@@ -0,0 +1,24 @@
+export const argTypes = {
+  external: {
+    name: 'external',
+    type: { name: 'boolean', required: false },
+    defaultValue: false,
+  },
+  href: {
+    name: 'href',
+    type: { name: 'string', required: false },
+    defaultValue: false,
+  },
+  target: {
+    type: 'select',
+    options: ['_self', '_blank', '_parent', '_top'],
+  },
+};
+
+export const componentArgs = {
+  external: false,
+  href: 'https://example.com',
+  default: 'Default Link',
```

Comment:
What does `default` mean in this context? `textContent`?

Reply:
`default` is a property defined in the storybook to refer to the `<slot>`, also it can be any name we want

Reply:
I think the `textContent` name makes a lot of sense so I will update it!